.github/workflows/build-ci.yml

```diff
@@ -36,16 +36,14 @@ jobs:
                 os:
                     - ubuntu-latest
                 mongodb:
-                    - '4.0'
-                    - '4.2'
-                    - '4.4'
                     - '5.0'
+                    - '6.0'
                 php:
                     - '8.1'
                     - '8.2'
         services:
             mysql:
-                image: mysql:5.7
+                image: mysql:8.0
```

Comment:
> While I was working I updated MySql/MongoDB versions in the dockerfile and github action file. I also dropped the support for mongoDB version 4 as starting from version 5 and above Mongosh is used instead of Mongo.

Sorry to ask, can you extract this CI changes into a dedicated PR for a separated review.

Reply:
I did extracted the CI changes into #2579. So you can remove this changes from the PR.

Reply:
@GromNaN Great! Thank you! I'll remove it soon, inshallah.